### ISSUE TITLE
Remove a TreeBuilder deprecation

### ISSUE THREAD

**Issue by arneee** [php-translation/symfony-bundle#275]
In Symfony 4.2 the following deprecation warning appears:
```
A tree builder without a root node is deprecated since Symfony 4.2 and will not be supported anymore in 5.0.
Hide context    Show trace
[▼
  "exception" => SilencedErrorContext {#3109 ▼
    +count: 2
    -severity: E_USER_DEPRECATED
    trace: {▼
      C:\...\sfphp\vendor\symfony\config\Definition\Builder\TreeBuilder.php:30 {▶}
      C:\...\sfphp\vendor\php-translation\symfony-bundle\DependencyInjection\Configuration.php:36 {▼
        › {
        ›     $treeBuilder = new TreeBuilder();
        ›     $root = $treeBuilder->root('translation');
      }
    }
  }
]
```

**Comment by egorsmkv** [comment]
Support this issue.

**Comment by egorsmkv** [comment]
@odolbeau made this a month ago https://github.com/odolbeau/symfony-bundle/commit/e7ec066e7e6565e85c6178599cfe688665fff0a9

**Comment by mpiot** [comment]
Is it possible to bump a new version that handle this changes ?

**Comment by crabnky** [comment]
+1

**Comment by bocharsky-bw** [comment]
Looks like it was fixed in https://github.com/php-translation/symfony-bundle/pull/273 for this bundle, though there's still no release for it

**Comment by bocharsky-bw** [comment]
Because the problem was fixed - closing it in favor of #287
